minecraft-ftb-fabric/minecraft-ftb-fabric.json

```diff
@@ -0,0 +1,133 @@
+{
+  "name": "minecraft-ftb-fabric",
+  "display": "FTB - Fabric",
+  "type": "minecraft-java",
+  "install": [
+    {
+      "type": "javadl",
+      "version": "${javaversion}"
+    },
+    {
+      "files": [
+        "https://api.modpacks.ch/public/modpack/${modpack_id}/${modpack_version}/server/linux"
+      ],
+      "type": "download"
+    },
+    {
+      "commands": [
+        "chmod u+x linux",
+        "./linux ${modpack_id} ${modpack_version} --auto --noscript"
```

Comment:
iirc, the wrapper file they download above actually includes those flags, so we don't need an explicit CLI arg here for it.

Reply:
You mean `./linux ${modpack_id} ${modpack_version} --auto --noscript` should not have `--auto --noscript`?  
In the other two FTB templates it is the same way, so it would require changing them there too.

Reply:
The `${modpack_id} ${modpack_version}` is not required in the call, because the binary already contains the ids.

Reply:
Changed to `"./linux --auto --noscript"`
Would you like me to change the other FTB templates (on another PR)?

Reply:
It does need to be `./linux ${modpack_id} ${modpack_version} --auto --noscript` or it will error with 

> Starting process: ./linux --auto --noscript
> Server installer version 22.1213.2347-linux-amd64 commit fba9792cf71889ce973a6a0846a194e3113e1678
>                       _                  _              _     
>                      | |                | |            | |    
> 
>  based on commit fba9792cf71889ce973a6a0846a194e3113e1678
> 
> Usage:
>   linux <modpackid> <versionid> - will install the modpack specified by <modpackid> with the version specified by <versionid>
>   linux <modpackid> - will install the modpack specified by <modpackid> with the latest version available
> 
>   _ __ ___   ___   __| |_ __   __ _  ___| | _____   ___| |__  
>  | '_ ` _ \ / _ \ / _` | '_ \ / _` |/ __| |/ / __| / __| '_ \ 
>  | | | | | | (_) | (_| | |_) | (_| | (__|   <\__ \| (__| | | |
>  |_| |_| |_|\___/ \__,_| .__/ \__,_|\___|_|\_\___(_)___|_| |_|
>                        | |                                    
>                        |_|                                    
>  modpacks.ch server downloader golang - build 22.1213.2347-linux-amd64Arguments:
> 2023/03/06 19:42:00 Cannot locate modpack via filename. Error: unable to parse filename: linux
>   --auto - Ask no questions, use defaults.
>   --path - Directory to install in. Default: current directory
>   --noscript - Skip creating start script. Default: false
>   --nojava - Skip downloading a compatible Adoptium JRE. Default: false
>   --threads - Number of threads to use for downloading. Default: cpucores * 2Do a full integrity check, even on files not changed by the update. integrityupdate assumed. Default: true
> 
>   --integrityupdate - Whether changed files should be overwritten with fresh copies when updating. Most useful when used with Auto. Default: false
>     Example: You changed config/test.cfg on your server from default. The modpack updates config/test.cfg - with this flag, it will assume you wish to overwrite with the latest version
>   --integrity -   --latest -   --verbose - Be a bit noisier on actions taken. Default: false
> Install latest, ignoring any version in the file name or arguments. Default: false
>   --curseforge - Specifies that pack is a Curseforge modpack
>   --help - This help"
>